src/main/connector/system/Windows/Windows.yaml

```diff
@@ -274,3 +274,26 @@ monitors:
           system.disk.io_time: $6
           system.disk.operation_time{disk.io.direction="read"}: $7
           system.disk.operation_time{disk.io.direction="write"}: $8
+  system:
+    simple:
+      sources:
+        # SerialNumber;Caption;Version;BuildNumber
+        osInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT SerialNumber, Caption, Version, BuildNumber FROM Win32_OperatingSystem
+        # Uptime (timestamp)
+        uptime:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT LastBootUpTime FROM Win32_OperatingSystem

```

Comment:
Use `SELECT SystemUpTime FROM Win32_PerfFormattedData_PerfOS_System` to get the system uptime (and not the date of the last reboot).